### PR TITLE
don't show vertical scroll bar in desktop sidebars

### PR DESCRIPTION
**Pull request by haileyok**
Fixes https://github.com/bluesky-social/social-app/issues/3246

## Why

As seen in this image, scrollbars on Windows in our left and right navs cause a pretty ugly layout. Generally this isn't as bad on macOS since the OS handles scrollbars a bit better, but they can also look bad there too.

img

## How

If we replace the current `View` that wraps these with a `ScrollView` and set the `showVerticalScrollIndicator` prop to `false`, these are not displayed, creating a better look.

## Before - Right Nav

Screenshot 2024-03-19 at 1 35 50 PM
Screenshot 2024-03-19 at 1 36 01 PM

## After - Right Nav

Screenshot 2024-03-19 at 1 35 41 PM
Screenshot 2024-03-19 at 1 36 33 PM
Screenshot 2024-03-19 at 1 36 37 PM

## Before - Left Nav

Screenshot 2024-03-19 at 1 39 00 PM
Screenshot 2024-03-19 at 1 39 06 PM

## After - Left Nav

Screenshot 2024-03-19 at 1 39 27 PM
Screenshot 2024-03-19 at 1 40 38 PM

### CODE REVIEW

**Comment by haileyok** [comment]
Mm, after looking at Twitter's layout, they do this a bit differently. Closing in favor of another option.

**Comment by adamatic-me** [comment]
oh, @haileyok  - somehow I missed in my search before making an issue related to this: https://github.com/bluesky-social/social-app/issues/3248#issuecomment-2004329588

TL;DR: I proposed only showing the scrollbar on hover, and also setting scrollbar-width: thin;
